### DISCUSSION TITLE
Entrar de nuevo a Teams

### DISCUSSION THREAD

**Discussion by GerardoCabreraH** [LaunchX-InnovaccionVirtual/CursoIntroPython#45] [Q&A] [answered]
Buenas tardes, mi nombre es Gerardo Cabrera y estoy en el programa de Innovacción Virtual, lo que pasa es que tuve que cerrar sesión por error y ya no puedo entrar a Teams, como se va a hacer para que pueda entrar de nuevo la sesión. Mi correo de acceso es este: launchx10957@innovaccion.mx

**Comment by ErickJimenez1826** [comment, marked as answer]
Deberias contactar a **contacto@innovaccion.mx** con tu cuenta y ID de registro

**Reply by GerardoCabreraH** [reply]
Nada mas tengo la cuenta, pero no tengo el ID del Registro

**Reply by EinarDvls** [reply]
el ID te lo mandan al correo personal que pusiste para inscribirte

**Reply by GerardoCabreraH** [reply]
Ya me lo resolvieron el contacto de innovacion, muchas gracias.
